fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/ManagedRdf.java

```diff
@@ -35,12 +41,25 @@
 
     private static final Model model = createDefaultModel();
 
+    private static final Set<URI> managedTypes = of(
```

Comment:
Do we understand what we mean by "managed" and "internal"? Do those qualities overlap?